tooling/lsp/src/requests/mod.rs

```diff
@@ -301,6 +301,7 @@ fn on_formatting_inner(
     state: &LspState,
     params: lsp_types::DocumentFormattingParams,
 ) -> Result<Option<Vec<lsp_types::TextEdit>>, ResponseError> {
+    let file_path = params.text_document.uri.to_file_path();
```

Comment:
Are you deliberately passing the path of the document being edited to `Config::read`, rather than the directory containing the document? I suppose it works, e.g. `/foo/bar/baz.nr` will try `/foo/bar/baz.nr/noirfmt.toml` first, then `/foo/bar/noirfmt.toml`, but it's a bit confusing.

Reply:
Ah, good point. I initially did it to make sure it checked parent paths but it makes sense to start with the parent.

Reply:
Done!